go.mod

```diff
@@ -6,29 +6,29 @@ toolchain go1.23.7
 
 require (
 	github.com/google/go-cmp v0.7.0
-	github.com/hashicorp/terraform-plugin-go v0.29.0-beta.1
+	github.com/hashicorp/terraform-plugin-go v0.29.0-beta.1.0.20250903123823-f82f646e532a
```

Comment:
This is based off the latest commit in https://github.com/hashicorp/terraform-plugin-go/pull/554, so it needs to be updated before merging 👍🏻